guides/common-features/render-patterns.md

```diff
@@ -6,7 +6,7 @@ patterns and when you might use them.
 ## `render/1` function clauses
 
 The most common pattern that is often used throughout this guide is the function clause pattern.
-Each function clause matches on the `:platform_id`, allowing each platform to define its own UI:
+Each function clause matches on the `:layout`, allowing each platform to define its own UI:
```

Comment:
I'm curious why `layout` instead of `format` here?

Reply:
@bcardarella Oops, good catch. Fixed!